libraries/lib-import-export/AcidizerTagSerialization.cpp

```diff
@@ -0,0 +1,33 @@
+#include "AcidizerTagSerialization.h"
+
+#include <rapidjson/document.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/writer.h>
+
+namespace LibImportExport
+{
+std::string AcidizerTagsToString(const LibFileFormats::AcidizerTags& tags)
+{
+   rapidjson::StringBuffer buffer;
+   rapidjson::Writer<rapidjson::StringBuffer> writer { buffer };
+   writer.StartObject();
+   writer.Key("bpm");
+   writer.Double(tags.bpm);
+   writer.Key("isOneShot");
+   writer.Bool(tags.isOneShot);
+   writer.EndObject();
+   return buffer.GetString();
+}
+
+LibFileFormats::AcidizerTags StringToAcidizerTags(const std::string& str)
+{
+   rapidjson::Document document;
+   document.Parse(str.c_str());
+   if (
+      !document.IsObject() || !document.HasMember("bpm") ||
+      !document.HasMember("isOneShot") || !document["bpm"].IsDouble() ||
+      !document["isOneShot"].IsBool())
+      return { 0., false };
```

Comment:
Maybe return `nullopt` instead.  Elsewhere you write a function taking `optional<AcidizerTags>`

Reply:
I agree, that's better.